src/assets/css/styles.css

```diff
@@ -1,3 +1,19 @@
+html {
+  font-size: 62.5%;
```

Comment:
why shrinking the `html` font-size and do a bigger font-size on the `body`? Could we remove the `font-size: 62.5%` from the html and set a `font-size` in px to the body and all of the children will inherit it?

Reply:
At this moment, If remove the `font-size: 62.5%` in the html will break alot of elements, because `em` and `rem` are been used in some elements.